silo/migrations/0040_auto_20180125_0149.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2018-01-25 09:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
Did you change the migration script manually? Because this import isn't used.

Reply:
Yes, makemigrations added migrations from other models i didn't want to change now. So i removed them

Reply:
ok. So, remove this unused import as well.